keyboards/exclusive/e6v2/le/le.c

```diff
@@ -21,12 +21,13 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
 	return process_record_user(keycode, record);
 }
 
+__attribute__ ((weak))
 void led_set_user(uint8_t usb_led) {
 	if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
-		DDRB |= (1 << 7); 
+		DDRB |= (1 << 7);
```

Comment:
Well, we don't need the DDRB for each here. .... it should only be called in a `keyboard_pre_init_kb` function

And also, it may be best to convert these to GPIO commands.

Reply:
Yep, I can do that — right after I turn this into `led_set_kb`.